tests/unit/network/test_graphql.py

```diff
@@ -159,6 +186,46 @@ def test_null_setter(pre_result, expected_result):
     assert post_result == expected_result
 
 
+@pytest.mark.parametrize(
+    'pre_result,'
+    'expected_result',
+    [
+        (
+            'foo',
+            'foo'
+        ),
+        (
+            [NULL_VALUE],
+            []
+        ),
+        (
+            {'nothing': NULL_VALUE},
+            {},
+        ),
+        (
+            TASK_PROXY_PREREQS.prerequisites,
+            TASK_PROXY_PREREQS.prerequisites
+        ),
+        (
+            [NULL_VALUE],
+            [],
+        )
```

Comment:
Duplicate of above
```suggestion
```

Reply:
Ah, yes ... I did delete a legitimate test..